src/routes/+layout.svelte

```diff
@@ -129,7 +129,7 @@
 			</div>
 		</svelte:fragment>
 		<svelte:fragment slot="sidebarLeft">
-			<div class="flex h-full flex-col py-[2vw] md:w-[6.25vw]">
+			<div class="flex h-full flex-col justify-between py-[2vw] md:w-[6.25vw]">
```

Comment:
Why `justify-between` ?
Fan of undefined heights?

Reply:
No better way ig